.github/workflows/vendor_gardener.yaml

```diff
@@ -0,0 +1,31 @@
+name: Vendor Gardener
```

Comment:
Maybe you could rename the file and the name field to something other than vendoring? (as we no longer have the vendor dir or make target named vendor)

Reply:
wdyt about something like "Sync Gardener Script Dependencies"

Reply:
No strong opinion. The above suggestion is ok. Alternatively, you can use something around make tidy as at the end the github action does this. Whatever you prefer.